build.gradle

```diff
@@ -124,6 +127,20 @@ if (hasProperty("buildScan")) {
 	}
 }
 
+// CRL schema extraction
+configurations {
+	cloudResourceSchema { transitive = false }
+}
+dependencies {
+	cloudResourceSchema group: 'bio.terra.cloud-resource-lib', name: 'cloud-resource-schema'
+}
+task unzipCloudResourceSchema(type: Copy) {
+	from zipTree(configurations.cloudResourceSchema.singleFile).matching {
+		include 'cloud_resources_uid.yaml'
+	}
+	into "${buildDir}/crlSchema"
```

Comment:
The thing with this is that now the yaml file depends on a build step and a new file in the local filesystem several directories up. It's not clear  how all our openAPI tools treat this, and it appears Swagger UI doesn't handle it well.

Reply:
I think this is a larger discussion at an MC terra level, which was where we decided to share yaml via artifactory. it's worth surfacing those issues, but I think it's out of scope for this PR

Reply:
sounds good. I want to try the other way to see if that's even viable. Don't know the URL I'd use for it though.